examples/generative/ddpm.py

```diff
@@ -0,0 +1,784 @@
+"""
+Title: Denoising Diffusion Probabilistic Model
+Author: [A_K_Nain](https://twitter.com/A_K_Nain)
+Date created: 2022/11/30
+Last modified: 2022/11/30
+Description: Generating images of flowers with denoising diffusion probabilistic models.
+"""
+
+"""
+## Introduction
```

Comment:
Please add the following to the introduction:

- State early that this example implements a specific paper, and briefly introduce the paper (and link to it).
- Link to the other keras.io diffusion example, and highlight some differences with the previous example.